### ISSUE TITLE
File level `ty` suppression comments

### ISSUE THREAD

**Issue by klonuo** [astral-sh/ty#1520]
### Summary

Hi,

I had a file with typing issue on several locations and thought to make file level comment to suppress it.

I tried to add the directive in the top of the file:
```
# ty: ignore[possibly-missing-attribute]
```
as such approach works for ruff, but it didn't work.

Reading the docs I couldn't find a solution either (besides editing config file).

I then read about `type:ignore` PEP 484 directive and that worked:

```
# type: ignore[possibly-missing-attribute]
```

Now I solved my problem and know the approach, but thought to write that maybe allowing also `ty: ignore` for file level suppression would be intuitive.

### Version

ty 0.0.1-alpha.25

**Comment by sharifhsn** [comment]
Closed by https://github.com/astral-sh/ruff/pull/24096

Repro (with `--error possibly-missing-attribute`):

```py
# ty: ignore[possibly-missing-attribute]

def flag() -> bool:
    return True

class Public:
    if flag():
        b = 2
    else:
        b: str

print(Public.b)
```

ty 0.0.24:

```
warning[unused-ignore-comment]: Unused `ty: ignore` directive
 --> foo.py:1:1
  |
1 | # ty: ignore[possibly-missing-attribute]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[possibly-missing-attribute]: Attribute `b` may be missing on class `Public`
  --> foo.py:12:7
   |
12 | print(Public.b)
   |       ^^^^^^^^

Found 2 diagnostics
```

ty 0.0.25:

```
All checks passed!
```
